Quick_Sort/Quick_Sort.js

```diff
@@ -1,27 +1,36 @@
-function Quick_Sort(array, low, high){
+//Description: Function to perform the Quick Sort algorithm on an array of numbers
+//Expected Output: returns the sorted array
```

Comment:
```suggestion
//Expected Output: returns the sorted array
```
move this to the beginning of the file